functions/admin-pages.php

```diff
@@ -83,19 +83,50 @@ function gnt_settings_page_content() {
  */
 function gnt_info_page_content() {
 	?><div class="wrap">
+
 		<h1><?php esc_attr_e( 'Information', 'get-notified' ); ?></h1>
-		<p>This is an open source project that I started to give developers the tools they need to create great notifications for a WordPress site. I welcome any help from developers who want to contribute :)</p>
-		<p>There are two main parts of the plugin: Hooks and Integrations. Hooks are basically the WordPress defined action hooks for any specific event. This can be when a post is published or when a theme is activated. Integrations are the tools used to notify people of these events, like email or slack. Developers can add new Hooks and/or Integrations to the plugin to perform the tasks they need.</p>
 
-		<a href="https://github.com/kjbenk/get-notified" target="_blank">Contribute - Read more</a> |
-		<a href="https://github.com/kjbenk/get-notified/issues" target="_blank">Report a Bug</a>
+		<p><?php esc_attr_e( 'Get Notified sends you notifications when certain events happen on your site.', 'get-notified' ); ?></p>
```

Comment:
Not a big deal, but please try to use the proper escaping function next time.  `esc_attr` is for an HTML attribute like `<p data-id="<?php echo esc_attr( $id ); ?>"></p>`, and the `esc_html` function is used for the actual HTML of the tag like`<p data-id="<?php echo esc_html( $content ); ?>"></p>`